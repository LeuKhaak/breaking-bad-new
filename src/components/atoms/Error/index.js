import React from "react";
import * as styles from "./styles.module.scss";

function Error({ notice }) {
  return <div className={styles.error}>{notice}</div>;
}

export default Error;
