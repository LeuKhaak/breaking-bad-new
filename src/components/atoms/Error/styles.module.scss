@import "../../../assets/styles/styles.scss";
.error {
  min-height: calc(100vh - 10rem);
  display: flex;
  justify-content: center;
  font-size: 1.5rem;
  font-weight: 500;
  color: $error;
  padding: 1rem 0;
}
