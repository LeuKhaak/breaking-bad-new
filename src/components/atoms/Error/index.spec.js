import React from "react";
import { shallow } from "enzyme";
import Error from ".";

let component = null;
const setComponent = (props) => shallow(<Error {...props} />);
beforeEach(() => {
  component = setComponent();
});

describe("Error testing:", () => {
  test("is Error contain text", () => {
    component = setComponent({
      notice: "Error",
    });
    expect(component).toMatchSnapshot();
  });
  test("is Error contain nothing", () => {
    component = setComponent({});
    expect(component).toMatchSnapshot();
  });
});
