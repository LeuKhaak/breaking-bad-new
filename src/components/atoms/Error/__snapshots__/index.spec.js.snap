// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error testing: is Error contain nothing 1`] = `<div />`;

exports[`Error testing: is Error contain text 1`] = `
<div>
  Error
</div>
`;
